templates/terraform/examples/base_configs/test_body.go.erb

```diff
@@ -0,0 +1,4 @@
+	return fmt.Sprintf(`
+<%= ctx[:content] -%>
+`,<% i = 0; loop do %><% break if i >= ctx[:count] %> prefix,<% i += 1 %><% end %>
```

Comment:
Just passing through- can you use [.times](https://ruby-doc.org/core-2.2.2/Integer.html#method-i-times) here?

Also, I looked at the generated code and it's not really a prefix, maybe something more generic like val would be better (since right now it's appearing in things like "address-%s" but it _could_ pop up anywhere in the string)

Reply:
Done

Reply:
Done